### ISSUE TITLE
feature-envy complains about accessing static properties of the same class

### ISSUE THREAD

**Issue by Nimelrian** [Glavin001/tslint-clean-code#37]
The feature-envy rule complains about this:
```ts
class Foo {
    private static constant = 5;

    // Method "doStuff" uses "Foo" more than its own class "Foo". Extract or Move Method from "doStuff" into "Foo". (no-feature-envy)
    public doStuff(num: number) {
        return Foo.constant + num;
    }
}
```

The static property obviously belongs to Foo, but the rule still complains.

**Comment by Glavin001** [comment]
Pull Requests welcome!

Applicable files:
- https://github.com/Glavin001/tslint-clean-code/blob/master/src/tests/NoFeatureEnvyRuleTests.ts
- https://github.com/Glavin001/tslint-clean-code/blob/master/src/noFeatureEnvyRule.ts

Please let me know if you have any questions. 😃

**Comment by Glavin001** [comment]
Published to v0.2.6
